README.md

```diff
@@ -115,3 +130,17 @@ To open a REPL with the executable scoped:
 ```bash
 make repl-exe
 ```
+
+## Be part of the community
+
+We would love to hear what you think of Juvix! Join our community:
+
+- Follow us on [Twitter](https://twitter.com/juvixlang)
+- Subscribe to our [newsletter](https://juvix.org/#TODO)
+
+<!-- TODO add links to discord, reddit, etc. -->
```

Comment:
Perhaps @awasunyin or @alexfmpe can help fill out this section?

Reply:
For now the subscribe feature is implemented on our website, let's just add a reference to the official website juvix.org and they'll be able to find the subscribe button there.